java/runtime/src/main/java/io/ray/runtime/config/RayConfig.java

```diff
@@ -130,6 +132,28 @@ public RayConfig(Config config) {
       nodeIp = NetworkUtil.getIpAddress(null);
     }
     this.nodeIp = nodeIp;
+    // Redis configurations.
```

Comment:
Why do you move this block of code upward?

Reply:
reverted